### PR TITLE
Reworked Logistic Regression description and other minor fixes

### PR DESCRIPTION
**Pull request by petit-pois**
WIP, I may push other fixes as I've read sections only up to "Quick Start Guide"

### CODE REVIEW

**Comment by petit-pois** [comment]
@ngama75 I've added you as optional reviewer

File: source-xor/index.html.md [hunk 1]
```diff
@@ -292,27 +291,44 @@ A training dataset consisting of k features $x_1,\dots,x_k$, each with N samples
...
-dataset <- read.csv("trainingSet.csv", header = FALSE)
-X <- dataset[,1:ncol(dataset) - 1]
-y <- dataset[,ncol(dataset)]
+data <- read.csv("data.csv", header = FALSE)
+X <- data[,1:ncol(data) - 1]
+y <- data[,ncol(data)]
 model <- glm(y ~ ., data = X, family=binomial())
 ```
 
-A training dataset consisting of k features $x_1, \dots, x_k$, each with N samples, and a dependent vector y of binary labels of size N are secret shared among the compute nodes. This operation computes a model $\theta_0, \dots,\theta_k$ such that the prediction $\theta_0 + \sum_{i=1}^{k} θ_i x_i$ has the same sign as the training vector y. The quality of the result is measured in terms of accuracy (percentage of good predictions), recall, and precision (more meaningful for rare events predictions).
```
Comment (petit-pois) [inline source-xor/index.html.md]: image

File: source-xor/index.html.md [hunk 2]
```diff
@@ -292,27 +291,44 @@ A training dataset consisting of k features $x_1,\dots,x_k$, each with N samples
...
+A training dataset consisting of $k$ features $x_1, \dots, x_k$, each with $n$ samples, and a dependent vector $y$ $\in \\{ 0, 1 \\}$ of binary labels of size $n$ are secret shared among the compute nodes. This operation computes a model $\theta_0, \dots,\theta_k$ such that the prediction vector $\hat y$ can be obtained as follow:
```
Comment (petit-pois) [inline source-xor/index.html.md]: image

Comment (ngama75) [inline source-xor/index.html.md]: il vaut mieux déplacer le $\in \\{ 0, 1 \\}$  apres le "of binary labels"

à la fin: 
the coefficients of the prediction vector haty can be obtained, elementwise, as

File: source-xor/index.html.md [hunk 3]
```diff
@@ -292,27 +291,44 @@ A training dataset consisting of k features $x_1,\dots,x_k$, each with N samples
...
+\hat y =
+\begin{cases}
+1 & \text{ if } \quad \theta_0 + \sum_{i=1}^{k} θ_i \cdot x_i > 0\\\\ 
+0 & \text{ otherwise} 
+\end{cases}
+$$
+
+The quality of the result is measured in terms of accuracy (percentage of good predictions), recall, and precision (more meaningful for rare events predictions).
+
+The probability of the predictions given the explanatory variables $X_{\lbrack n \times k \rbrack}$ and model $\theta$ can be computed by feeding the log-odds $t = \theta_0 + \sum_{i=1}^{k} θ_i \cdot x_i$ into the logistic function $\sigma(t) = \frac{1}{1 + e^{-t}}$
```
Comment (ngama75) [inline source-xor/index.html.md]: est-ce qu'on peut garder "features" plutot que "explanatory variables"?

File: source-xor/index.html.md [hunk 4]
```diff
@@ -498,7 +514,7 @@ Then connect to the analyst portal and login using the credentials provided by y
 
 ## Connecting to the Analyst Platform
 
-Once your **XOR Machines** are up and running, you need to login to the **Analyst Platform**. The XOR Analyst Platform is hosted on the analyst portal. Upon authorization by Inpher, you will receive your long credentials, so use those to login to the Analyst Platform.
+Once your **XOR Machines** are up and running, you need to login to the **Analyst Platform**. The XOR Analyst Platform is hosted on the analyst portal. Upon authorization by Inpher, you will receive your loging credentials, use them to connect to the Analyst Platform.
```
Comment (ngama75) [inline source-xor/index.html.md]: login credentials?

File: source-xor/index.html.md (hunk 2 above)
Comment (Picnixz) [reply source-xor/index.html.md]: you can also write `$y \in \{0, 1\}^n$` instead

File: source-xor/index.html.md [hunk 5]
```diff
@@ -256,7 +255,7 @@ sum(error_sq)
 ```
 Residual Sum of Squares (RSS) is the sum of the squares of residuals. The smaller the RSS, the tighter the fit of the model to the data.  
 
-For input matrix X (m x n) and a model vector (n), the matrix-vector product first yields a vector P (m). The elements of P represent the prediction of y and residual sum of squares is computed as follows: $\sum_{i=1}^{m} (y_i - p_i)^2$ 
+For input matrix $X$ of size $\lbrack n \times k \rbrack$ and a model vector $\theta$ of size $k$, the matrix-vector product yields a vector $\hat y$ of size $n$, representing the prediction of $y$. The residual sum of squares is computed as follows: $\sum_{i=1}^{n} (y_i - \hat y_i)^2$
```
Comment (Picnixz) [inline source-xor/index.html.md]: When dealing with matrices (and not vectors), I would rather use the word `shape` instead of size and not square brackets, but rather curly ones: `for input matrix $X$ of shape $(n, k)$` (or `$(n \times k)$` if you want).

File: source-xor/index.html.md (hunk 4 above)
Comment (petit-pois) [reply source-xor/index.html.md]: oups thanks

File: source-xor/index.html.md (hunk 2 above)
Comment (petit-pois) [reply source-xor/index.html.md]: v2 
image

Comment (Picnixz) [reply source-xor/index.html.md]: I was perhaps unclear: usually you would write `a vector $y \in \{0, 1\}^n$ of binary labels are secret shared ...` or `a vector $y$ of $n$ binary labels in $\{0, 1\}$ are secret shared..` so that you don't think that the labels themselves are vectors. I personally prefer the first solution as it really gives you the space in which `y` lives.

File: source-xor/index.html.md [hunk 6]
```diff
@@ -266,22 +265,22 @@ For input matrix X (m x n) and a model vector (n), the matrix-vector product fir
...
 y = data[:,-1]
 
 model = LinearRegression()
 model.fit(X, y)
 ```
 ```r
-dataset <- read.csv("inputData.csv", header = FALSE)
-X <- dataset[,1:ncol(dataset) - 1]
-y <- dataset[,ncol(dataset)]
+data <- read.csv("data.csv", header = FALSE)
```
Comment (shiftux) [inline source-xor/index.html.md]: i actually like the terminology of `dataset` 
we had this discussion with the new frontend developer and kind of agree to keep `dataset` and `data sources` everywhere

**Comment by shiftux** [comment]
LGTM, some minor things. 
thanks for going through it!

File: source-xor/index.html.md (hunk 2 above)
Comment (petit-pois) [reply source-xor/index.html.md]: I also prefer the first version but I changed as of @ngama75 comment above

File: source-xor/index.html.md (hunk 6 above)
Comment (petit-pois) [reply source-xor/index.html.md]: so I change data to dataset everywhere ? I changed this snippet to "uniformize" the code examples

**Comment by petit-pois** [comment]
I've finished to go through the public documentation, please do an extra pass of review

File: source-xor/index.html.md (hunk 6 above)
Comment (shiftux) [reply source-xor/index.html.md]: yup would make sense IMO. but this was discussed in a meeting with nothing on the spot. 
other opinions welcome!

Comment (petit-pois) [reply source-xor/index.html.md]: done, please review
